docs/pkp/authMethods/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
file not needed since we are hardcoding this in the sidebar, the category file is needed when we tell docusaurus to auto-gen the folder structure